app/models/cis2_info.rb

```diff
@@ -44,6 +44,10 @@ def can_prescribe_pgd?
     role_code == NURSE_ROLE
   end
 
+  def can_prescribe_psd?
```

Comment:
I'm wondering actually if we need to rename `can_prescribe_pgd?` to `can_supply_pgd?` since that covers both the prescribing and the supplying of the vaccine (i.e. what nurses do).